### ISSUE TITLE
ImportError: cannot import name MarkdownLangMap

### ISSUE THREAD

**Issue by pkkid** [sublimelsp/LSP-pyright#146]
I am unable to get this working in sublime-text-3. After install LSP and LSP-pyright, I get the following in console:

```
reloading plugin LSP-pyright.commands
reloading plugin LSP-pyright.plugin
Traceback (most recent call last):
  File "/opt/sublime_text/sublime_plugin.py", line 125, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.3/importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/opt/sublime_text/sublime_plugin.py", line 1199, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "/home/pkkid/.config/sublime-text-3/Installed Packages/LSP-pyright.sublime-package/plugin.py", line 1, in <module>
ImportError: cannot import name MarkdownLangMap
reloading plugin LSP.boot
```

I have the following:
* sublime-text-3 v3.2.2, Build 3211
* ubuntu 22.04
* python 3.10.3

**Comment by jfcherng** [comment]
I can confirm this.

**Comment by jfcherng** [comment]
Fixed in [1.1.154](https://github.com/sublimelsp/LSP-pyright/releases/tag/1.1.154).

**Comment by pkkid** [comment]
Well that was super fast. Thanks so much!
